float64 sr
float64 status
float64 vel
float64 xr
float64 yr
float64 zr
float64 grad
float64 lr